Makefile

```diff
@@ -96,11 +98,11 @@ build-catalog-md:
 
 # build the CNF test binary
 build-cnf-tests:
-	PATH=${PATH}:${GOBIN} ginkgo build -ldflags "-X github.com/test-network-function/test-network-function/test-network-function.GitCommit=${GIT_COMMIT}" ./test-network-function 
+	PATH=${PATH}:${GOBIN} ginkgo build -ldflags "-X github.com/test-network-function/test-network-function/test-network-function.GitCommit=${GIT_COMMIT} -X github.com/test-network-function/test-network-function/test-network-function.GitRelease=${GIT_RELEASE} -X github.com/test-network-function/test-network-function/test-network-function.GitPreviousRelease=${GIT_PREVIOUS_RELEASE}" ./test-network-function 
```

Comment:
we not just use "commit=" and "version=" ?
why we need previous release as well ?

Reply:
The previous version is used to give the last release of a given unrelease build. For instance, currently it will say unreleased, post 3.0.0.

Reply:
I think my comment was not clear. 
what you're passing after -X is key=value; the key is very long. we need only 
``` -X commit = ${GIT_COMMIT} Release = ${GIT_RELEASE} -X PreviousRelease = ${}.. ``

Reply:
This is the name of the static variable object in the executable, not sure how to shorten it. I tried the short but ended up needing a obj decoder to find out the working name. See:
[deliedit@cnfcert-local cnftest]$ go tool nm test-network-function/test-network-function.test  | grep -i gitCommit
  f05540 D github.com/test-network-function/test-network-function/test-network-function.GitCommit
  b450a0 R github.com/test-network-function/test-network-function/test-network-function.GitCommit.str